src/Compilers/CSharp/Portable/Syntax/SyntaxNormalizer.cs

```diff
@@ -11,6 +11,44 @@
 
 namespace Microsoft.CodeAnalysis.CSharp.Syntax
 {
+    internal static class Extensions
```

Comment:
I'll move to a separate file or find an existing suitable class if you agree with the change here.